chain/consensus/filcns/upgrades.go

```diff
@@ -1557,8 +1557,8 @@ func UpgradeActorsV10(ctx context.Context, sm *stmgr.StateManager, cache stmgr.M
 	root cid.Cid, epoch abi.ChainEpoch, ts *types.TipSet) (cid.Cid, error) {
 
 	// TODO migration
-	// - the init actor state to include the (empty) installed actors field
-	// - state tree migration to v5
+	// - state tree migration to v5; must set predictable addresses for all account actors
+	// - create EAM actor
```

Comment:
We want to create Eth address `0x00..00` too as part of the migration. This address is defined by spec. cc @Stebalien

Reply:
will add a comment here.

Reply:
done.